src/pages/Detail/DetailC.scss

```
@@ -14,3 +14,17 @@
     }
   }
 }
+
+.empty-ele {
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
```

```suggestion
  width: 100vw;
  height: 100vh;
  position: fixed;
  top: 0;
  bottom: 0;
  right: 0;
  left: 0;
}
```
fixed에 top,left,bottom,right이 다 0으로 offset을 부여하면, viewport를 꽉 채운 효과가 자연스럽게 발생해서 width, height 값을 따로 부여해줄 필요가 없습니다!